domain-services/ifix-department-entity-service/src/main/java/org/egov/service/DepartmentHierarchyLevelService.java

```diff
@@ -0,0 +1,41 @@
+package org.egov.service;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.egov.config.IfixDepartmentEntityConfig;
+import org.egov.repository.DepartmentHierarchyLevelRepository;
+import org.egov.validator.DepartmentHierarchyLevelValidator;
+import org.egov.web.models.DepartmentHierarchyLevelRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class DepartmentHierarchyLevelService {
+
+    @Autowired
+    private DepartmentHierarchyLevelValidator validator;
+
+    @Autowired
+    private DepartmentHierarchyLevelEnrichmentService enricher;
+
+    @Autowired
+    private IfixDepartmentEntityConfig departmentEntityConfig;
```

Comment:
@pintu-eGov 
Remove any unused objects in the next PR.

Reply:
done.